### PR TITLE
[#13395] Improve RPC stats section appearance

### PR DESCRIPTION
**Pull request by Rende11**
#13395

### Summary
Added some styles to the RPC stats screen

### Testing notes
Tested with emulators:
- iPhone 11 (Simulator version 12.4)
- Pixel 5 API 30 (Android 11 x86)

#### Platforms
- Android
- iOS

status: ready

Screenshot 2022-06-30 at 18 15 37
Screenshot 2022-06-30 at 18 16 00

### CODE REVIEW

**Comment by ghost** [comment]
Hey @Rende11, and thank you so much for making your first pull request in status-react! :heart: Please help us make your experience better by filling out this brief questionnaire https://goo.gl/forms/uWqNcVpVz7OIopXg2

**Comment by Parveshdhull** [comment]
hi @Rende11, Thank you very much for PR, looks really good.

File: src/status_im/ui/screens/rpc_usage_info.cljs [hunk 1]
```diff
@@ -85,36 +86,45 @@
 (defn stats-table [{:keys [total filtered-total stats]}]
   [quo.react-native/scroll-view
    {:style {:padding-horizontal 8}}
-   [quo.react-native/view
-    {:style {:flex-direction :row
-             :justify-content :space-between}}
-    [quo.core/text "TOTAL"]
-    [quo.core/text (str filtered-total " of " total)]]
+   [quo.react-native/view {:style {:margin-bottom 2}}
+    [quo.react-native/view
```
Comment (Parveshdhull) [inline src/status_im/ui/screens/rpc_usage_info.cljs]: why extra view is needed why don't add margin to second view

File: src/status_im/ui/screens/rpc_usage_info.cljs [hunk 2]
```diff
@@ -85,36 +86,45 @@
...
-       ^{:key (str k v)}
-       [quo.react-native/view
-        {:style {:flex-direction :row
-                 :align-items     :center}}
-        [quo.core/text {:style {:flex 1}} k]
-        [quo.core/text  {:style {:margin-left 16}} v]]))])
+     (doall
+      (for [[k v] stats]
+        ^{:key (str k v)}
+        [quo.react-native/view
```
Comment (Parveshdhull) [inline src/status_im/ui/screens/rpc_usage_info.cljs:101]: ` [quo.react-native/view` can be replaced with `[:<>`

File: src/status_im/ui/screens/rpc_usage_info.cljs [hunk 3]
```diff
@@ -85,36 +86,45 @@
...
+      (i18n/label :t/rpc-usage-filtered-total {:filtered-total filtered-total :total total})]]]
    (when (seq stats)
-     (for [[k v] stats]
-       ^{:key (str k v)}
-       [quo.react-native/view
-        {:style {:flex-direction :row
-                 :align-items     :center}}
-        [quo.core/text {:style {:flex 1}} k]
-        [quo.core/text  {:style {:margin-left 16}} v]]))])
+     (doall
```
Comment (Parveshdhull) [inline src/status_im/ui/screens/rpc_usage_info.cljs]: Please can you explain why `doall` is needed?

Comment (Rende11) [reply src/status_im/ui/screens/rpc_usage_info.cljs]: I thought about rendering lazy sequences - https://old.purelyfunctional.tv/guide/reagent/#lazy

Comment (flexsurfer) [reply src/status_im/ui/screens/rpc_usage_info.cljs]: doall needed only if you have deref inside

**Review by Parveshdhull — APPROVED** [review]

**Comment by qoqobolo** [comment]
@Rende11 thanks for the PR and for your contribution!
Ready for merge.

**Comment by Rende11** [comment]
@qoqobolo  Merging is blocked - e2e tests are failed

**Review by flexsurfer — APPROVED** [review]

**Comment by qoqobolo** [comment]
Hey @Rende11, PR has been successfully merged.
Just FYI: that e2e failure is not related to PR, so it was ignored.

**Pull request merged**